package.xml

```diff
@@ -2,8 +2,17 @@
 <?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="2">
   <name>rcpputils</name>
-  <version>0.1.0</version>
+  <version>0.2.0</version>
```

Comment:
Any reason to start from 0.2.0?

Reply:
Not really, I just committed the original empty package as 0.1.0